it/lessons/basics/enum.md

```diff
@@ -9,14 +9,36 @@ Un insieme di algoritmi per enumerare le collezioni.
 
 ## Enum
 
-Il modulo `Enum` contiene oltre un centinaio di funzioni per lavorare con le collezioni di cui abbiamo parlato nella scorsa lezione.
+Il modulo `Enum` contiene oltre un 70 funzioni per lavorare con le collezioni di cui abbiamo parlato nella scorsa lezione.
+Tuttle le collezione di cui abbiamo imparato nella lezione precendente, eccetto le tuple. sono enumerazioni.
```

Comment:
Typo here: `collezione` (singular) should be `collezioni` (plural).

Reply:
fixed in efe71e5a7017e4a5602d069b368c9c56d9880e4b